### PR TITLE
Pal 710 ecr push

### PR DESCRIPTION
**Pull request by ac74475**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -28,292 +28,305 @@
...
-                        sh 'mvn -s $MAVEN_SETTINGS install'
+            stage('SonarQube Analysis') {
+                dir('Palisade-services') {
+                    container('docker-cmds') {
+                        withCredentials([string(credentialsId: "${env.SQ_WEB_HOOK}", variable: 'SONARQUBE_WEBHOOK'),
+                                         string(credentialsId: "${env.SQ_KEY_STORE_PASS}", variable: 'KEYSTORE_PASS'),
+                                         file(credentialsId: "${env.SQ_KEY_STORE}", variable: 'KEYSTORE')]) {
+                            configFileProvider([configFile(fileId: "${env.CONFIG_FILE}", variable: 'MAVEN_SETTINGS')]) {
+                                withSonarQubeEnv(installationName: 'sonar') {
+                                        sh "mvn -s ${MAVEN_SETTINGS} org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar -Dsonar.projectKey=Palisade-Services-${GIT_BRANCH_NAME} -Dsonar.projectName=Palisade-Services-${GIT_BRANCH_NAME} -Dsonar.webhooks.project=$SONARQUBE_WEBHOOK -Djavax.net.ssl.trustStore=$KEYSTORE -Djavax.net.ssl.trustStorePassword=$KEYSTORE_PASS"
```
Comment (developer01189998819991197253) [inline Jenkinsfile:245]: this doesnt work and just creates a project key called Palisade-Services-
revert back to the bulky if statement

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]
small change

**Review by dev930018 — DISMISSED** [review]

**Review by developer01189998819991197253 — DISMISSED** [review]

**Review by pd104923 — DISMISSED** [review]

**Review by dev930018 — DISMISSED** [review]

**Review by developer01189998819991197253 — DISMISSED** [review]

**Review by pd104923 — DISMISSED** [review]

**Review by developer01189998819991197253 — DISMISSED** [review]

**Review by dev-958 — DISMISSED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -28,292 +28,319 @@
...
+                                            "--set global.persistence.classpathJars.aws.volumeHandle=${VOLUME_HANDLE_CLASSPATH_JARS} " +
+                                            "--set global.redisClusterEnabled=true " +
+                                            "--set global.redis.install=false " +
+                                            "--set global.redis-cluster.install=true " +
+                                            "--namespace ${HELM_DEPLOY_NAMESPACE}", returnStatus: true) == 0) {
+                                        echo("successfully deployed")
+                                    } else {
+                                        error("Build failed because of failed maven deploy")
+                                    }
+                                    if (FEATURE_BRANCH == "true") {
```
Comment (dev-958) [inline Jenkinsfile:336]: I'm kind of thinking that this should be IS_PR == "true"? Am I right?

**Review by dev-958 — DISMISSED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (ac74475) [reply Jenkinsfile:336]: So either should work as it only gets into that code block if it is a PR or not a feature branch.

Comment (pd104923) [reply Jenkinsfile:336]: I think this is correct. The `IS_PR` flag is already true due to the initial if statement in this stage and we only want to create the namespace and delete the namespace if `FEATURE_BRANCH` is true.

**Review by pd104923 — DISMISSED** [review]

**Review by dev930018 — DISMISSED** [review]

**Review by dev-958 — APPROVED** [review]

**Pull request merged**